app/scripts/app.js

```diff
@@ -56,12 +69,13 @@ angular.module('yourStlCourts').config(function($stateProvider, $urlRouterProvid
         court: {value: undefined}
       },
       resolve: {
-        court: function ($stateParams,Courts) {
-          if ($stateParams.courtId != ""){
-            return Courts.findById($stateParams.courtId)
-          }else{
-            return null;
-          };
+        court: function ($stateParams, Courts, Errors) {
+          if ($stateParams.courtId == "") {
```

Comment:
Try to never use "==" in JavaScript. It will coerce variables into different types that may give unintended consequences. "Falsy" in JS includes: null, undefined, false, [], "", etc. Also, our linting is setup for single quotes, not double quotes like you have here.

this should work just as well and covers more cases:
```
if (!$stateParams.courtId)
```